tabs.js

```diff
@@ -0,0 +1,92 @@
+$(function () {
+  const board_id = localStorage.getItem("board_id");
+  const token = localStorage.getItem("trello_token");
+  const statsContainer = document.getElementById("cardWrapper");
+  let listsOfData;
+
+  // locaser
+
+  const htmlLoader = () => {
+    const markup = `
+<div class="loader">
+      <div class="line"></div>
+      <div class="line"></div>
+      <div class="line"></div>
+      <div class="line"></div>
+    </div>
+
+`;
+    statsContainer.insertAdjacentHTML("afterbegin", markup);
+  };
+  // clear the loader
+
+  const clearUI = () => {
+    statsContainer.innerHTML = "";
+  };
+
+  // html for the card
+  const htmlCard = ({ name, noOfCounts }) => {
+    return `
+<div class="col-md-6 mt-1" style="width:50%;">
```

Comment:
Can you remove the styling of the data? Like, remove any cards, padding, margin, anything.
Let's just have plain text:
```
Wishlist: 2
Applied: 4
```
Just simple text. I'll work on the CSS myself later.